SDL/Quintuplet.h

```diff
@@ -2338,6 +2338,7 @@ namespace SDL
                 int nTotQ = alpaka::atomicOp<alpaka::AtomicAdd>(acc, &nTotalQuintupletsx, occupancy);
                 rangesInGPU.quintupletModuleIndices[i] = nTotQ;
                 rangesInGPU.indicesOfEligibleT5Modules[nEligibleT5Modules] = i;
+                rangesInGPU.quintupletModuleOccupancy[i] = occupancy;
```

Comment:
Hi Andres, Is this value not filled in the previous code, so you fill it here for completeness? Or it was filled somewhere else? Thanks!

Reply:
Hi @YonsiG, this value was previously not being filled or used anywhere. This is why in the `createQuintupletsInGPUv2` kernel the maximum occupancy was being checked in this other way. But we should be using this array, similar to how it's done in other parts of the code.

For example, for triplets this is how it works:
https://github.com/SegmentLinking/TrackLooper/blob/6d818d4b77fccef161d798c9e6a228ea6936d417/SDL/Triplet.h#L1430
https://github.com/SegmentLinking/TrackLooper/blob/6d818d4b77fccef161d798c9e6a228ea6936d417/SDL/Triplet.h#L1345

Reply:
Thanks Andres!